circuit-runtime/src/commonMain/kotlin/com/slack/circuit/runtime/Navigator.kt

```diff
@@ -15,6 +15,9 @@ public interface Navigator {
   /** Returns current top most screen of backstack, or null if backstack is empty. */
   public fun peek(): Screen?
 
+  /** Returns the current back stack. */
+  public fun peekBackStack(): List<Screen>
```

Comment:
We'd have to make the `resetRoot` overload composable, which probably isn't great.

Reply:
It doesn't because we use it to manipulate records together. Can think on it and see if any alternatives come up before the release, but this doesn't seem bad.

Reply:
We could remove the `resetRoot` overload and just rely on the basic `Boolean` version for now. It's mostly a nice to have anyway

Reply:
👍